File: problem-test/src/main/java/com/dudgns/problemtest/entity/carManagement/CarCategoryEntity.java
package com.dudgns.problemtest.entity.carManagement;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "car_category")
public class CarCategoryEntity {
    @Id
    @Column(name = "category_type")
    private Integer categoryType;

    @Column(name = "category_name")
    private String categoryName;
}

File: problem-test/commons/project-dudgns-data-entity/src/main/kotlin/dudgns/com/backend/commons/data/entity/problemTest/CarCategoryEntity.kt
package dudgns.com.backend.commons.data.entity.problemTest

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "car_category")
class CarCategoryEntity {
    @Id
    @Column(name = "category_type")
    var categoryType: Int? = null

    @Column(name = "category_name")
    var categoryName: String? = null
}